fi/features.html

```diff
@@ -0,0 +1,458 @@
+                <li><strong>Differenssiyhtälöt</strong></li>
+                <li><strong>Yhtälöryhmät</strong></li>
+                <li><strong>Diofantoksen yhtälöt</strong> <a href="http://www.sympygamma.com/input/?i=diophantine%28x%2A%2A2%20-%204%2Ax%2Ay%20%2B%208%2Ay%2A%2A2%20-%203%2Ax%20%2B%207%2Ay%20-%205%29">x<sup>2</sup> &#8722; 4xy + 8y<sup>2</sup> &#8722; 3x + 7y = 5</a>, <a href="http://www.sympygamma.com/input/?i=diophantine%282%2Ax%20%2B%203%2Ay%20-%205%29">2x + 3y = 5</a></li>
+            </ul>
+        </div>
+    </div>
+
+    <div class="dropdown">
+        <h3>Kombinatoriikka</h3>
+        <div>
+            <p><a href="http://docs.sympy.org/latest/modules/combinatorics/index.html">Dokumentaatio</a></p>
+            <ul>
+                <li><strong>Permutaatiot</strong></li>
+                <li><strong>Kombinaatiot</strong></li>
+                <li><strong>Ositukset</strong></li>
+                <li><strong>Aliryhmät</strong></li>
+                <li><strong>Permutaatioryhmät</strong>: Polyhedral, Rubik, Symmetric, ...</li>
+                <li><strong>Prüfer ja Gray - koodit</strong></li>
+            </ul>
+        </div>
+    </div>
+
+    <div class="dropdown">
+        <h3>Diskreetti matematiikka</h3>
+        <div>
+            <ul>
+                <li><strong>Binomikertoimet</strong></li>
+                <li><strong>Summat</strong></li>
+                <li><strong>Tulot</strong></li>
+                <li><strong>Lukuteoria:</strong> alkulukujen generointi, alkulukutestit, kokonaislukuhajotelmat, diofantoksen yhtälöt...</li>
+                <li><strong>Loogiset lauseet</strong></li>
+            </ul>
+        </div>
+    </div>
+
+    <div class="dropdown">
+        <h3>Matriisit</h3>
+        <div>
+            <p><a href="http://docs.sympy.org/latest/modules/matrices/index.html">Dokumentaatio</a></p>
+            <ul>
+                <li><strong>Perusaritmetiikka</strong></li>
+                <li><strong>Ominaisarvot ja ominaisvektorit</strong></li>
+                <li><strong>Determinantit</strong></li>
+                <li><strong>Inversiot</strong></li>
+                <li><strong>Ratkaiseminen</strong></li>
+                <li><strong>Abstraktit lausekkeet</strong></li>
+            </ul>
+        </div>
+    </div>
+
+    <div class="dropdown">
+        <h3>Geometrinen algebra</h3>
+        <div>
+            <p><a href="http://docs.sympy.org/latest/modules/galgebra/index.html">Dokumentaatio</a></p>
+        </div>
+    </div>
+
+    <div class="dropdown">
+        <h3>Geometria</h3>
+        <div>
+            <p><a href="http://docs.sympy.org/latest/modules/geometry.html">Dokumentaatio</a></p>
+            <ul>
+                <li><strong>pisteet, suorat, säteet, segmentit, ellipsit, ympyrät,
+                    monikulmiot, ...</strong></li>
+                <li><strong>Leikkaus</strong></li>
+                <li><strong>Tangentti</strong></li>
```

Comment:
I think that "Tangency" could be more accurately translated as "Sivuaminen". One could also say "Tangentit" or "Sivuajat".